test/lib/mocked-device-api.ts

```diff
@@ -133,20 +132,25 @@ function buildRoutes(appManager: ApplicationManager): Router {
 	return router;
 }
 
+const originalNetGetAll = networkManager.getAllByAppId;
+const originalVolGetAll = volumeManager.getAllByAppId;
+const originalSvcGetStatus = serviceManager.getStatus;
 function setupStubs() {
-	stub(ServiceManager.prototype, 'getStatus').resolves(STUBBED_VALUES.services);
-	stub(NetworkManager.prototype, 'getAllByAppId').resolves(
-		STUBBED_VALUES.networks,
-	);
-	stub(VolumeManager.prototype, 'getAllByAppId').resolves(
-		STUBBED_VALUES.volumes,
-	);
+	// @ts-expect-error Assigning to a RO property
+	networkManager.getAllByAppId = () => Promise.resolve(STUBBED_VALUES.networks);
```

Comment:
These would be nicer imo as
```suggestion
	networkManager.getAllByAppId = async () => STUBBED_VALUES.networks;
```